pkg/errors/error.go

```diff
@@ -49,6 +49,11 @@ var (
 	// after some wait time
 	TemporaryOutOfSync = fmt.Errorf(
 		"temporary out of sync, reconcile after some time")
+	// RequeueOnSuccess is to indicate the error isn't really an error
+	// but more of a marker that the status check will be performed
+	// after some wait time after the specified duration
+	RequeueOnSuccess = fmt.Errorf(
+		"resource creation successful but reconcile after some time")
```

Comment:
I don't believe you need the above error. You can just pass `nil` to `requeue.NeededAfter()`...

Reply:
ok, done.